src/Illuminate/Validation/Rules/Password.php

```diff
@@ -270,36 +270,39 @@ public function passes($attribute, $value)
     {
         $this->messages = [];
 
-        $validator = Validator::make($this->data, [
-            $attribute => 'string|min:'.$this->min,
-        ], $this->validator->customMessages, $this->validator->customAttributes);
+        $validator = Validator::make(
+            $this->data,
+            [$attribute => 'string|min:'.$this->min],
+            $this->validator->customMessages,
+            $this->validator->customAttributes
+        )->after(function ($validator) use ($attribute, $value) {
+            if (! is_string($value)) {
```

Comment:
@PovilasKorop why was this added? Isn't `string` baked into the validation rules above?

Reply:
@crynobone a question for you here